{"expand":"renderedFields,names,schema,operations,editmeta,changelog,versionedRepresentations","id":"40871","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871","key":"GUI-2442","fields":{"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803},"timespent":null,"customfield_13100":null,"project":{"self":"https://eucalyptus.atlassian.net/rest/api/2/project/11301","id":"11301","key":"GUI","name":"Management Console","avatarUrls":{"48x48":"https://eucalyptus.atlassian.net/secure/projectavatar?pid=11301&avatarId=10011","24x24":"https://eucalyptus.atlassian.net/secure/projectavatar?size=small&pid=11301&avatarId=10011","16x16":"https://eucalyptus.atlassian.net/secure/projectavatar?size=xsmall&pid=11301&avatarId=10011","32x32":"https://eucalyptus.atlassian.net/secure/projectavatar?size=medium&pid=11301&avatarId=10011"}},"customfield_11000":null,"customfield_11001":null,"fixVersions":[],"aggregatetimespent":null,"customfield_10310":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10209","value":"CentOS 6","id":"10209"}],"resolution":{"self":"https://eucalyptus.atlassian.net/rest/api/2/resolution/5","id":"5","description":"All attempts at reproducing this issue failed, or not enough information was available to reproduce the issue. Reading the code produces no clues as to why this behavior would occur. If more information appears later, please reopen the issue.","name":"Cannot Reproduce"},"customfield_11401":null,"customfield_11400":null,"customfield_10104":[],"customfield_10500":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10403","value":"Not Applicable","id":"10403"},"customfield_10302":null,"customfield_10303":null,"customfield_10105":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10101","value":"No","id":"10101"},"customfield_10700":null,"customfield_10304":null,"customfield_10701":null,"customfield_10702":null,"customfield_10306":null,"customfield_10703":null,"resolutiondate":"2016-02-17T09:18:21.934-0600","customfield_10704":null,"customfield_10309":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10204","value":"KVM","id":"10204"}],"customfield_10705":null,"workratio":-1,"lastViewed":null,"watches":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/GUI-2442/watchers","watchCount":4,"isWatching":false},"created":"2016-02-12T04:19:02.554-0600","customfield_12000":null,"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/3","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/major.svg","name":"Major","id":"3"},"customfield_12400":null,"customfield_12201":null,"customfield_10102":null,"customfield_12600":"{}","customfield_10300":null,"labels":[],"customfield_10301":null,"aggregatetimeoriginalestimate":null,"timeestimate":null,"customfield_11901":null,"versions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/16405","id":"16405","description":"Eucalyptus 4.2.1","name":"4.2.1","archived":false,"released":true,"releaseDate":"2015-12-07"}],"issuelinks":[],"assignee":null,"updated":"2016-03-23T07:10:55.298-0500","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"components":[],"timeoriginalestimate":null,"customfield_13000":null,"description":"In HPE Singapore demo cloud we have installed Euca 4.2.1\r\n\r\nThe setup is following\r\n\r\n1 host - CLC\r\n1 host - UFS / Console / Walrus\r\n1 host - CC / SC\r\n2 hosts - NC\r\n\r\nWe are not able to login to the console. It keeps timing out. The log suggest console is unable to establish HTTPS connection with the UFS. We are not sure how to fix the problem.\r\n\r\nThe installation is a fresh install. \r\n\r\nconsole.ini\r\n\r\n{code}\r\n[app:main]\r\nuse = egg:eucaconsole\r\n\r\nufshost = 192.168.6.5 \r\nufsport = 8773\r\n\r\nhelp.url=https://support.eucalyptus.com/hc/en-us\r\nsupport.url=\r\n\r\nlog.useractions = false\r\n\r\naws.enabled = True\r\naws.default.region = us-east-1\r\n\r\nstatic.cache.duration = 43200\r\n\r\nbrowser.password.save = false\r\n\r\nfile.uploads.enabled = true\r\n\r\nconnection.ssl.validation = false\r\n\r\n\r\ncloudformation.url.whitelist = http://*, https://*\r\n\r\nconnection.debug = true \r\nconnection.retries = 2\r\n\r\npyramid.reload_templates = false\r\npyramid.debug_authorization = false\r\npyramid.debug_notfound = false\r\npyramid.debug_routematch = false\r\npyramid.default_locale_name = en\r\npyramid.includes =\r\n    pyramid_beaker\r\n    pyramid_chameleon\r\n    pyramid_layout\r\n\r\nsession.type = cookie\r\nsession.key = eucaconsole_session\r\nsession.keyini = /etc/eucaconsole/session-keys.ini\r\nsession.httponly = true\r\nsession.secure = true\r\nsession.timeout = 1800\r\nsession.cookie_expires = 43200\r\n\r\ncache.memory = dogpile.cache.pylibmc\r\ncache.memory.url = 127.0.0.1:11211\r\ncache.short_term.expire = 60\r\ncache.default_term.expire = 300\r\ncache.long_term.expire = 3600\r\ncache.extra_long_term.expire = 43200\r\ncache.images.disable = false\r\n\r\n[server:main]\r\nuse = egg:gunicorn#main\r\nhost = 127.0.0.1\r\nport = 8888\r\nworkers = 4\r\nworker_class = eucaconsole.gunicorn.ggevent.GeventWorker\r\n\r\n\r\n[loggers]\r\nkeys = root\r\n\r\n[handlers]\r\nkeys = screen_debug, logfile\r\n\r\n[formatters]\r\nkeys = generic, eucaformat\r\n\r\n[logger_root]\r\nhandlers = logfile\r\nlevel = INFO\r\n\r\n[handler_screen_debug]\r\nclass = StreamHandler\r\nargs = (sys.stdout,)\r\nlevel = NOTSET\r\nformatter = generic\r\n\r\n[handler_logfile]\r\nclass = handlers.RotatingFileHandler\r\nargs = ('/var/log/eucaconsole.log', 'a', 'maxBytes=1000000', 'backupCount=10')\r\nlevel = NOTSET\r\nformatter = eucaformat\r\n\r\n[formatter_generic]\r\nformat = %(asctime)s %(levelname)-5.5s [%(name)s][%(threadName)s] %(message)s\r\n\r\n[formatter_eucaformat]\r\nformat = %(asctime)-15s %(levelname)s %(message)s\r\ndatefmt = %Y-%m-%d %H:%M:%S\r\n{code}\r\n\r\nnginx.conf\r\n\r\n{code}\r\n# nginx.conf\r\n\r\n# user nginx;\r\nworker_processes 4;\r\npid /var/run/eucaconsole/nginx.pid;\r\n\r\nevents {\r\n    worker_connections 2048;\r\n    # multi_accept on;\r\n}\r\n\r\nhttp {\r\n    ### Basic Settings\r\n    ##\r\n\r\n    sendfile on;\r\n    tcp_nopush on;\r\n    tcp_nodelay on;\r\n    keepalive_timeout 15;\r\n    types_hash_max_size 2048;\r\n    # server_tokens off;\r\n\r\n    # server_names_hash_bucket_size 64;\r\n    # server_name_in_redirect off;\r\n\r\n    # include /etc/nginx/mime.types;\r\n    default_type application/octet-stream;\r\n\r\n    ### Logging Settings\r\n    ##\r\n\r\n    access_log /var/log/eucaconsole_nginx_access.log;\r\n    error_log /var/log/eucaconsole_nginx_error.log;\r\n\r\n    ### Gzip Settings\r\n    ##\r\n\r\n    gzip on;\r\n    # gzip_disable \"msie6\";\r\n\r\n    ### Virtual Host Configs\r\n    ##\r\n\r\n    server {\r\n        listen 80 default;\r\n        server_name ~^(?<domain>.+)$;\r\n        rewrite     ^ https://$domain$request_uri? permanent;\r\n    }\r\n\r\n    server {\r\n        listen 443 ssl;\r\n        server_name ~^(?<domain>.+)$;\r\n\r\n        # SSL configuration\r\n        ssl_protocols TLSv1 TLSv1.1 TLSv1.2;\r\n        ssl_ciphers \"ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:ECDHE-RSA-DES-CBC3-SHA:ECDHE-ECDSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:AES:CAMELLIA:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!aECDH:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA\";\r\n        ssl_prefer_server_ciphers on;\r\n        ssl_certificate /etc/eucaconsole/console.crt;\r\n        ssl_certificate_key /etc/eucaconsole/console.key;\r\n        add_header \"X-UA-Compatible\" \"IE=Edge\";\r\n        add_header Strict-Transport-Security \"max-age=31536000; includeSubDomains\";\r\n        # end of SSL configuration\r\n\r\n        location /static/.* {\r\n            # NOTE: Set path to static folder\r\n            root                    /path/to/eucaconsole/static;\r\n            expires                 30d;\r\n            add_header              Cache-Control public;\r\n            access_log              off;\r\n        }\r\n\r\n        location / {\r\n            proxy_set_header        Host $http_host;\r\n            proxy_set_header        X-Real-IP $remote_addr;\r\n            proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;\r\n            proxy_set_header        X-Forwarded-Proto $scheme;\r\n\r\n            client_max_body_size    5g;\r\n            client_body_buffer_size 128k;\r\n            proxy_connect_timeout   30s;\r\n            proxy_send_timeout      90s;\r\n            proxy_read_timeout      90s;\r\n            proxy_buffering on;\r\n            proxy_buffer_size 128k;\r\n            proxy_buffers 4 256k;\r\n            proxy_busy_buffers_size 256k;\r\n            proxy_temp_file_write_size 512k;\r\n            # NOTE: Specify Eucalyptus Management Console hosts/ports here\r\n            # Multiple console hosts are supported\r\n            proxy_pass http://127.0.0.1:8888;\r\n            # proxy_pass http://127.0.0.1:9999;\r\n            proxy_redirect          off;\r\n        }\r\n\r\n\r\n    }\r\n\r\n    # include /etc/nginx/conf.d/*.conf;\r\n    # include /etc/nginx/sites-enabled/*;\r\n}\r\n{code}\r\n\r\nThe log suggest following:\r\n\r\n{code}\r\nhost;x-amz-date;x-amz-security-token\r\n60dc8c81f7f3e72187d97d1ab341dbb59494ffcb2d73e15815c3f91aad12c620\r\n2016-02-12 18:14:21 DEBUG StringToSign:\r\nAWS4-HMAC-SHA256\r\n20160212T101421Z\r\n20160212/168/192/aws4_request\r\n82fd403a3482421b54decbfdba8ab9541179a8034bd77ffc46c15dd25c77047b\r\n2016-02-12 18:14:21 DEBUG Signature:\r\n42bbf68c64c32cc5bd6d59befebba90a0a1d46f13b7bc1a8e5aaac1dbc7fb837\r\n2016-02-12 18:14:21 DEBUG Final headers: {'Content-Length': '87', 'X-Amz-Date': '20160212T101421Z', 'User-Agent': 'Boto/2.38.0 Python/2.6.6 Linux/2.6.32-573.18.1.el6.x86_64', 'Host': '192.168.6.5:8773', 'X-Amz-Security-Token': u'ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==', 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8', 'Authorization': u'AWS4-HMAC-SHA256 Credential=AKIAAY7ZNMWPGAKGOPJW/20160212/168/192/aws4_request,SignedHeaders=host;x-amz-date;x-amz-security-token,Signature=42bbf68c64c32cc5bd6d59befebba90a0a1d46f13b7bc1a8e5aaac1dbc7fb837'}\r\n2016-02-12 18:14:51 DEBUG encountered SSLError exception, reconnecting\r\n2016-02-12 18:14:51 DEBUG establishing HTTPS connection: host=192.168.6.5, kwargs={'port': 8773, 'timeout': 30}\r\n2016-02-12 18:14:52 DEBUG Token: ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==\r\n2016-02-12 18:14:52 DEBUG CanonicalRequest:\r\nPOST\r\n/services/compute/\r\n\r\nhost:192.168.6.5:8773\r\nx-amz-date:20160212T101452Z\r\nx-amz-security-token:ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==\r\n\r\nhost;x-amz-date;x-amz-security-token\r\n60dc8c81f7f3e72187d97d1ab341dbb59494ffcb2d73e15815c3f91aad12c620\r\n2016-02-12 18:14:52 DEBUG StringToSign:\r\nAWS4-HMAC-SHA256\r\n20160212T101452Z\r\n20160212/168/192/aws4_request\r\n248e80ebd0f852870112eb1b40c6ca71eaf687890509a35f98674211d7fccef8\r\n2016-02-12 18:14:52 DEBUG Signature:\r\na79f432f0f9deb7e031fcd2a89cf5f909f2f6738b8864d01633bcc335094350b\r\n2016-02-12 18:14:52 DEBUG Final headers: {'Content-Length': '87', 'X-Amz-Date': '20160212T101452Z', 'User-Agent': 'Boto/2.38.0 Python/2.6.6 Linux/2.6.32-573.18.1.el6.x86_64', 'Host': '192.168.6.5:8773', 'X-Amz-Security-Token': u'ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==', 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8', 'Authorization': u'AWS4-HMAC-SHA256 Credential=AKIAAY7ZNMWPGAKGOPJW/20160212/168/192/aws4_request,SignedHeaders=host;x-amz-date;x-amz-security-token,Signature=a79f432f0f9deb7e031fcd2a89cf5f909f2f6738b8864d01633bcc335094350b'}\r\n2016-02-12 18:15:22 DEBUG encountered SSLError exception, reconnecting\r\n2016-02-12 18:15:22 DEBUG establishing HTTPS connection: host=192.168.6.5, kwargs={'port': 8773, 'timeout': 30}\r\n2016-02-12 18:15:22 DEBUG Token: ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==\r\n2016-02-12 18:15:22 DEBUG CanonicalRequest:\r\nPOST\r\n/services/compute/\r\n\r\nhost:192.168.6.5:8773\r\nx-amz-date:20160212T101522Z\r\nx-amz-security-token:ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==\r\n\r\nhost;x-amz-date;x-amz-security-token\r\n60dc8c81f7f3e72187d97d1ab341dbb59494ffcb2d73e15815c3f91aad12c620\r\n2016-02-12 18:15:22 DEBUG StringToSign:\r\nAWS4-HMAC-SHA256\r\n20160212T101522Z\r\n20160212/168/192/aws4_request\r\n8c13a676806e5b3b957536f291633c2c3791fad097945342776c0acfb670fbc6\r\n2016-02-12 18:15:22 DEBUG Signature:\r\n885fa04febf3de435bc7c016180f6997338431a27fdb6f4bbc5f762abba05b31\r\n2016-02-12 18:15:22 DEBUG Final headers: {'Content-Length': '87', 'X-Amz-Date': '20160212T101522Z', 'User-Agent': 'Boto/2.38.0 Python/2.6.6 Linux/2.6.32-573.18.1.el6.x86_64', 'Host': '192.168.6.5:8773', 'X-Amz-Security-Token': u'ZXVjYQAByynNzX+s1nELTfM0Lv9rhvSPVdMvbMJWuvvBvnxPSesRg5Z5W/oYTyDrp+XD/lc14yhpe44LXZsHmaLpCbEWi2i53wj8ykhw4kAQ2/VNUmrKOUT/SaUtYuXB8E1HTwMaMymwe5fhMTzbKnhZK95UiwX1czdt4DRFRc+b9iqGJ0XNfuImLQe3UpzJ635ZRxUx2XuTDdJQG8Vj2FPTCYBnk8XdAkZUVY/JvsP/yFmYLRRWJppcW45Pjw==', 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8', 'Authorization': u'AWS4-HMAC-SHA256 Credential=AKIAAY7ZNMWPGAKGOPJW/20160212/168/192/aws4_request,SignedHeaders=host;x-amz-date;x-amz-security-token,Signature=885fa04febf3de435bc7c016180f6997338431a27fdb6f4bbc5f762abba05b31'}\r\n2016-02-12 18:15:52 DEBUG encountered SSLError exception, reconnecting\r\n2016-02-12 18:15:52 DEBUG establishing HTTPS connection: host=192.168.6.5, kwargs={'port': 8773, 'timeout': 30}\r\n2016-02-12 18:15:54 ERROR Error handling request\r\nTraceback (most recent call last):\r\n  File \"/usr/lib/python2.6/site-packages/gunicorn/workers/async.py\", line 45, in handle\r\n    self.handle_request(listener, req, client, addr)\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/gunicorn/ggevent.py\", line 154, in handle_request\r\n    super(GeventWorker, self).handle_request(*args)\r\n  File \"/usr/lib/python2.6/site-packages/gunicorn/workers/async.py\", line 93, in handle_request\r\n    respiter = self.wsgi(environ, resp.start_response)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/router.py\", line 251, in __call__\r\n    response = self.invoke_subrequest(request, use_tweens=True)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/router.py\", line 227, in invoke_subrequest\r\n    response = handle_request(request)\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/tweens.py\", line 102, in __call__\r\n    response = self.handler(request)\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/tweens.py\", line 58, in tween\r\n    response = handler(request)\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/tweens.py\", line 47, in tween\r\n    response = handler(request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/tweens.py\", line 21, in excview_tween\r\n    response = handler(request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/router.py\", line 161, in handle_request\r\n    response = view_callable(context, request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/config/views.py\", line 586, in __call__\r\n    return view(context, request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/config/views.py\", line 319, in attr_view\r\n    return view(context, request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/config/views.py\", line 295, in predicate_wrapper\r\n    return view(context, request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/config/views.py\", line 345, in rendered_view\r\n    result = view(context, request)\r\n  File \"/usr/lib/python2.6/site-packages/pyramid/config/views.py\", line 467, in _class_requestonly_view\r\n    response = getattr(inst, attr)()\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/views/login.py\", line 146, in handle_login\r\n    return self.handle_euca_login()\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/views/login.py\", line 179, in handle_euca_login\r\n    session['supported_platforms'] = self.get_account_attributes(['supported-platforms'])\r\n  File \"/usr/lib/python2.6/site-packages/eucaconsole/views/__init__.py\", line 279, in get_account_attributes\r\n    attributes = conn.describe_account_attributes(attribute_names=attribute_names)\r\n  File \"/usr/lib/python2.6/site-packages/boto/ec2/connection.py\", line 4426, in describe_account_attributes\r\n    [('item', AccountAttribute)], verb='POST')\r\n  File \"/usr/lib/python2.6/site-packages/boto/connection.py\", line 1170, in get_list\r\n    response = self.make_request(action, params, path, verb)\r\n  File \"/usr/lib/python2.6/site-packages/boto/connection.py\", line 1116, in make_request\r\n    return self._mexe(http_request)\r\n  File \"/usr/lib/python2.6/site-packages/boto/connection.py\", line 1030, in _mexe\r\n    raise ex\r\nSSLError: The read operation timed out\r\n{code}\r\n\r\nNot sure what special thing needs to be done on a fresh installation of Euca 4.2.1 to make Euca management console work.\r\n\r\nCheers,\r\nDeependra","customfield_13200":null,"customfield_11300":null,"customfield_11500":null,"timetracking":{},"customfield_10005":null,"customfield_10401":null,"customfield_10600":null,"customfield_12900":null,"attachment":[],"aggregatetimeestimate":null,"summary":"cannot login to the console","creator":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"subtasks":[],"reporter":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"aggregateprogress":{"progress":0,"total":0},"customfield_10000":"6_*:*_1_*:*_0_*|*_10000_*:*_1_*:*_449959390","customfield_12100":"0|i03mjz:","customfield_10001":"2016-02-12T08:05:22.861-0600","customfield_10200":null,"customfield_10002":null,"customfield_10003":null,"customfield_12500":null,"customfield_10400":null,"customfield_11600":["dkavanagh(dkavanagh)","dmccue(dmccue)","dshekhawat(dshekhawat)","kgill(kgill)"],"customfield_10116":null,"environment":null,"customfield_10117":null,"customfield_11800":null,"customfield_11802":null,"duedate":null,"customfield_11805":null,"progress":{"progress":0,"total":0},"customfield_11806":null,"comment":{"comments":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99194","id":"99194","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"body":"A simple HTTPS api call against the UFS works just fine using euca2ools (DescribeImages)\r\n\r\n{code}\r\n2016-02-12 18:19:57,522 DEBUG   DescribeImages parsed arguments: {'all': False, 'filters': None, 'security_token': None, 'url': None, 'region': None, 'ImageId': [], 'show_empty_fields': False, 'Owner': None, 'key_id': None, 'secret_key': None, 'ExecutableBy': None}\r\n2016-02-12 18:19:57,522 INFO    Config finding region option user\r\n2016-02-12 18:19:57,522 DEBUG   Config   section localhost\r\n2016-02-12 18:19:57,522 DEBUG   Config   section *\r\n2016-02-12 18:19:57,522 INFO    Config   novalue for user\r\n2016-02-12 18:19:57,522 INFO    Config finding global option max-retries\r\n2016-02-12 18:19:57,522 INFO    Config   novalue for max-retries\r\n2016-02-12 18:19:57,522 INFO    Config finding global option timeout\r\n2016-02-12 18:19:57,522 INFO    Config   novalue for timeout\r\n2016-02-12 18:19:57,522 INFO    Config finding region option verify-ssl\r\n2016-02-12 18:19:57,522 DEBUG   Config   section localhost\r\n2016-02-12 18:19:57,523 DEBUG   Config   section *\r\n2016-02-12 18:19:57,523 INFO    Config   novalue for verify-ssl\r\n2016-02-12 18:19:57,523 WARNING EC2 added fake region name undefined-19083\r\n2016-02-12 18:19:57,523 DEBUG   HmacV4Auth using auth info from environment\r\n2016-02-12 18:19:57,523 INFO    DescribeImages parameters: {'Action': 'DescribeImages', 'Version': '2014-06-15', 'Owner.1': 'self'}\r\n2016-02-12 18:19:57,523 DEBUG   DescribeImages sending flattened params as form data\r\n2016-02-12 18:19:57,525 INFO    EC2 sending request (attempt 1 of 3)\r\n2016-02-12 18:19:57,531 DEBUG   HmacV4Auth payload hash: ee99a9d5d6a8a5a894c5e722aa9e5ceb80b16e911f0d268004df78f60e3f7479\r\n2016-02-12 18:19:57,531 DEBUG   HmacV4Auth scope: 20160212/undefined-19083/ec2/aws4_request\r\n2016-02-12 18:19:57,531 DEBUG   HmacV4Auth canonical URI: /services/compute\r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth canonical query: \r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth canonical headers: ['accept:*/*', 'accept-encoding:gzip, deflate', 'connection:keep-alive', 'content-length:53', 'content-type:application/x-www-form-urlencoded', 'host:192.168.6.5:8773', 'x-amz-content-sha256:ee99a9d5d6a8a5a894c5e722aa9e5ceb80b16e911f0d268004df78f60e3f7479', 'x-amz-date:20160212T101957Z']\r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth signed headers: accept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date\r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth canonical request: 'POST\\n/services/compute\\n\\naccept:*/*\\naccept-encoding:gzip, deflate\\nconnection:keep-alive\\ncontent-length:53\\ncontent-type:application/x-www-form-urlencoded\\nhost:192.168.6.5:8773\\nx-amz-content-sha256:ee99a9d5d6a8a5a894c5e722aa9e5ceb80b16e911f0d268004df78f60e3f7479\\nx-amz-date:20160212T101957Z\\n\\naccept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date\\nee99a9d5d6a8a5a894c5e722aa9e5ceb80b16e911f0d268004df78f60e3f7479'\r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth string to sign: 'AWS4-HMAC-SHA256\\n20160212T101957Z\\n20160212/undefined-19083/ec2/aws4_request\\nbc2219e8a42e624315370bbdf7ec566910aeaddfcca87549b9d117dd0c9af01a'\r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth signature: 5ecbc13b09fdcd81586b0b91c5e9000d504a4022c505402bae4dd9a8b59a4e69\r\n2016-02-12 18:19:57,532 DEBUG   HmacV4Auth signed headers: accept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date\r\n2016-02-12 18:19:57,532 DEBUG   EC2 request method: POST\r\n2016-02-12 18:19:57,532 DEBUG   EC2 request url:    https://192.168.6.5:8773/services/compute\r\n2016-02-12 18:19:57,532 DEBUG   EC2 request header: Accept: */*\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: Accept-Encoding: gzip, deflate\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: Authorization: AWS4-HMAC-SHA256 Credential=AKIAAKQF77BVPTB4IWNV/20160212/undefined-19083/ec2/aws4_request, SignedHeaders=accept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date, Signature=5ecbc13b09fdcd81586b0b91c5e9000d504a4022c505402bae4dd9a8b59a4e69\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: Connection: keep-alive\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: Content-Length: 53\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: Content-Type: application/x-www-form-urlencoded\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: Host: 192.168.6.5:8773\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: User-Agent: euca2ools/3.3.0 (CPython 2.6.6; Linux 2.6.32-573.18.1.el6.x86_64; x86_64) requestbuilder/0.4.0 requests/2.6.0\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: X-Amz-Content-SHA256: ee99a9d5d6a8a5a894c5e722aa9e5ceb80b16e911f0d268004df78f60e3f7479\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request header: X-Amz-Date: 20160212T101957Z\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request data:   Action: DescribeImages\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request data:   Owner.1: self\r\n2016-02-12 18:19:57,533 DEBUG   EC2 request data:   Version: 2014-06-15\r\n/usr/lib/python2.6/site-packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html\r\n  InsecureRequestWarning)\r\n2016-02-12 18:19:57,577 DEBUG   EC2 response time: 0.044 seconds\r\n2016-02-12 18:19:57,578 INFO    EC2 response status: 200\r\n2016-02-12 18:19:57,578 DEBUG   EC2 response header: content-length: 925\r\n2016-02-12 18:19:57,578 DEBUG   EC2 response header: content-type: application/xml; charset=UTF-8\r\n2016-02-12 18:19:57,578 DEBUG   EC2 response header: date: Fri, 12 Feb 2016 10:19:57 UTC\r\n2016-02-12 18:19:57,578 DEBUG   DescribeImages -- response content --\r\n<DescribeImagesResponse xmlns=\"http://ec2.amazonaws.com/doc/2014-06-15/\"><requestId>282abc60-505e-4fab-8ca3-bd197fe72064</requestId><imagesSet><item><imageId>emi-6ce37e4b</imageId><imageLocation>eucalyptus-service-image-1.48/eucalyptus-service-image.raw.manifest.xml</imageLocation><imageState>available</imageState><imageOwnerId>000425215120</imageOwnerId><isPublic>false</isPublic><productCodes/><architecture>x86_64</architecture><imageType>machine</imageType><platform/><name>eucalyptus-service-image-v1.48</name><rootDeviceType>instance-store</rootDeviceType><rootDeviceName>/dev/sda</rootDeviceName><blockDeviceMapping/><virtualizationType>hvm</virtualizationType><tagSet><item><key>provides</key><value>imaging,loadbalancing,database</value></item><item><key>type</key><value>eucalyptus-service-image</value></item><item><key>version</key><value>1.48</value></item></tagSet></item></imagesSet></DescribeImagesResponse>\r\n2016-02-12 18:19:57,578 DEBUG   DescribeImages -- end of response content --\r\n2016-02-12 18:19:57,578 INFO    DescribeImages parameters: {'Action': 'DescribeImages', 'Version': '2014-06-15', 'ExecutableBy.1': 'self'}\r\n2016-02-12 18:19:57,578 DEBUG   DescribeImages sending flattened params as form data\r\n2016-02-12 18:19:57,579 INFO    EC2 sending request (attempt 1 of 3)\r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth payload hash: 131f3f0eec21b55872d60421962811430f3ea3e3ebefe8c0326bd23f92643a8d\r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth scope: 20160212/undefined-19083/ec2/aws4_request\r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth canonical URI: /services/compute\r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth canonical query: \r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth canonical headers: ['accept:*/*', 'accept-encoding:gzip, deflate', 'connection:keep-alive', 'content-length:60', 'content-type:application/x-www-form-urlencoded', 'host:192.168.6.5:8773', 'x-amz-content-sha256:131f3f0eec21b55872d60421962811430f3ea3e3ebefe8c0326bd23f92643a8d', 'x-amz-date:20160212T101957Z']\r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth signed headers: accept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date\r\n2016-02-12 18:19:57,579 DEBUG   HmacV4Auth canonical request: 'POST\\n/services/compute\\n\\naccept:*/*\\naccept-encoding:gzip, deflate\\nconnection:keep-alive\\ncontent-length:60\\ncontent-type:application/x-www-form-urlencoded\\nhost:192.168.6.5:8773\\nx-amz-content-sha256:131f3f0eec21b55872d60421962811430f3ea3e3ebefe8c0326bd23f92643a8d\\nx-amz-date:20160212T101957Z\\n\\naccept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date\\n131f3f0eec21b55872d60421962811430f3ea3e3ebefe8c0326bd23f92643a8d'\r\n2016-02-12 18:19:57,580 DEBUG   HmacV4Auth string to sign: 'AWS4-HMAC-SHA256\\n20160212T101957Z\\n20160212/undefined-19083/ec2/aws4_request\\n14658689517cf5d59c5386c7f502ebbfd7eadfcc58b9d935b604ae02656f7e01'\r\n2016-02-12 18:19:57,580 DEBUG   HmacV4Auth signature: 5089747990d1f9a4708ffbf684eef014c96d2a8964f4d6d59d384ddeeaf17002\r\n2016-02-12 18:19:57,580 DEBUG   HmacV4Auth signed headers: accept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request method: POST\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request url:    https://192.168.6.5:8773/services/compute\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Accept: */*\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Accept-Encoding: gzip, deflate\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Authorization: AWS4-HMAC-SHA256 Credential=AKIAAKQF77BVPTB4IWNV/20160212/undefined-19083/ec2/aws4_request, SignedHeaders=accept;accept-encoding;connection;content-length;content-type;host;x-amz-content-sha256;x-amz-date, Signature=5089747990d1f9a4708ffbf684eef014c96d2a8964f4d6d59d384ddeeaf17002\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Connection: keep-alive\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Content-Length: 60\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Content-Type: application/x-www-form-urlencoded\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: Host: 192.168.6.5:8773\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: User-Agent: euca2ools/3.3.0 (CPython 2.6.6; Linux 2.6.32-573.18.1.el6.x86_64; x86_64) requestbuilder/0.4.0 requests/2.6.0\r\n2016-02-12 18:19:57,580 DEBUG   EC2 request header: X-Amz-Content-SHA256: 131f3f0eec21b55872d60421962811430f3ea3e3ebefe8c0326bd23f92643a8d\r\n2016-02-12 18:19:57,581 DEBUG   EC2 request header: X-Amz-Date: 20160212T101957Z\r\n2016-02-12 18:19:57,581 DEBUG   EC2 request data:   Action: DescribeImages\r\n2016-02-12 18:19:57,581 DEBUG   EC2 request data:   ExecutableBy.1: self\r\n2016-02-12 18:19:57,581 DEBUG   EC2 request data:   Version: 2014-06-15\r\n2016-02-12 18:19:57,590 DEBUG   EC2 response time: 0.008 seconds\r\n2016-02-12 18:19:57,590 INFO    EC2 response status: 200\r\n2016-02-12 18:19:57,590 DEBUG   EC2 response header: content-length: 169\r\n2016-02-12 18:19:57,590 DEBUG   EC2 response header: content-type: application/xml; charset=UTF-8\r\n2016-02-12 18:19:57,590 DEBUG   DescribeImages -- response content --\r\n<DescribeImagesResponse xmlns=\"http://ec2.amazonaws.com/doc/2014-06-15/\"><requestId>513900d9-e7d4-4097-8acc-a1ce8464fa1e</requestId><imagesSet/></DescribeImagesResponse>\r\n2016-02-12 18:19:57,590 DEBUG   DescribeImages -- end of response content --\r\nIMAGE\temi-6ce37e4b\teucalyptus-service-image-1.48/eucalyptus-service-image.raw.manifest.xml\t000425215120\tavailable\tprivate\tx86_64\tmachine\t\t\t\tinstance-store\thvm\t\r\nTAG\timage\temi-6ce37e4b\tprovides\timaging,loadbalancing,database\r\nTAG\timage\temi-6ce37e4b\ttype\teucalyptus-service-image\r\nTAG\timage\temi-6ce37e4b\tversion\t1.48\r\n{code}\r\n\r\nNot sure why the Eucalyptus management console is not able to make same type of API call against HTTPS endpoint ??","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"created":"2016-02-12T04:21:31.224-0600","updated":"2016-02-12T04:21:31.224-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99195","id":"99195","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"body":"Nothing really comes to mind. Here's what I observe (in case it helps).\r\nIt looks like authorization succeeded (though I don't see it in the log) because the failure is in the login after auth where we try to fetch account attributes.\r\nIt looks like the right host/port are being used. It looks like the service path is set right \"/services/compute/\".\r\nFWIW, the auth call doesn't use boto, but the call that fails does (as does the rest of the app).","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"created":"2016-02-12T08:05:22.861-0600","updated":"2016-02-12T08:05:22.861-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99203","id":"99203","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=kgill","name":"kgill","key":"kgill","accountId":"557058:d51f06c5-6e24-45a0-83f2-493f2c8eac6e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue"},"displayName":"Kamal Gill","active":true,"timeZone":"America/Los_Angeles"},"body":"[~dshekhawat]: The console.ini looks pretty much the same as the default (at https://github.com/eucalyptus/eucaconsole/blob/master/conf/console.default.ini) except for the connection.debug setting.  Does login fail when connection.debug = false ?","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=kgill","name":"kgill","key":"kgill","accountId":"557058:d51f06c5-6e24-45a0-83f2-493f2c8eac6e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/8ed235fc95ad0f38552ce9617bdbf985?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dkgill%26avatarId%3D18200%26noRedirect%3Dtrue"},"displayName":"Kamal Gill","active":true,"timeZone":"America/Los_Angeles"},"created":"2016-02-12T12:58:19.502-0600","updated":"2016-02-12T12:58:19.502-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99233","id":"99233","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"body":"Hi [~kgill], Its strange that everything is working fine now after I set the connection.debug = false. Earlier it was giving me the same result. I am not sure if the console relies on a network configuration to be loaded. In the case of failure I had not configured the network.json.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"created":"2016-02-14T22:33:18.370-0600","updated":"2016-02-14T22:33:18.370-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99453","id":"99453","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"body":"[~dshekhawat], should we close this?","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"created":"2016-02-17T07:46:39.204-0600","updated":"2016-02-17T07:46:39.204-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99458","id":"99458","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"body":"Sounds good to me. Thank you\r\n\r\nSent from a mobile device\r\n\r\n","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"created":"2016-02-17T09:15:15.200-0600","updated":"2016-02-17T09:15:15.200-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/99477","id":"99477","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"body":"I can confirm that in order to reproduce this issue (I reproduced this twice in lab) you need to have a cloud without a network.json configured. Usually that can happen when you are installing a new cloud and before configuring the network try to access the user console. This access will fail in a very weird way suggesting something is wrong but not giving any further details. The root cause is that network.json is not configured. Perhaps a user friendly log message or something similar could be beneficial. ","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dshekhawat","name":"dshekhawat","key":"dshekhawat","accountId":"557058:dc10d216-39f8-4d84-a5ff-7d4daf8a44ed","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/7cda271db1cec24c198fdfb42ec8a8ae?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddshekhawat%26avatarId%3D15400%26noRedirect%3Dtrue"},"displayName":"Deependra Shekhawat","active":false,"timeZone":"Asia/Kolkata"},"created":"2016-02-17T21:23:44.355-0600","updated":"2016-02-17T21:23:44.355-0600"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/100439","id":"100439","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dmccue","name":"dmccue","key":"dmccue","accountId":"557058:ed3db3e7-1778-45e1-a9ca-da74c4afc860","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"David McCue","active":true,"timeZone":"Europe/London"},"body":"I encountered this during the DixonsCPW production install, some feedback that the network.json file is missing would assist with troubleshooting (Even in the logs)","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dmccue","name":"dmccue","key":"dmccue","accountId":"557058:ed3db3e7-1778-45e1-a9ca-da74c4afc860","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"David McCue","active":true,"timeZone":"Europe/London"},"created":"2016-03-17T06:27:22.360-0500","updated":"2016-03-17T06:27:22.360-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/100440","id":"100440","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"body":"[~dmccue], Deependra suggested a helpful log message would go a long way here. Help me understand more. Isn't the network.json file part of normal configuration for Edge mode? In what situations is that not present? Would using the cloud before the networking is fully configured be normal?\r\nWhat type of error does the console report in this case? If we show \"No response from host\" as we do when we cannot reach the CLC, that would seem appropriate. ","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"created":"2016-03-17T07:58:03.675-0500","updated":"2016-03-17T07:58:03.675-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/100606","id":"100606","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dmccue","name":"dmccue","key":"dmccue","accountId":"557058:ed3db3e7-1778-45e1-a9ca-da74c4afc860","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"David McCue","active":true,"timeZone":"Europe/London"},"body":"During the dixons install, we still havn't got the floating IP address range and so can't construct the network.json file.  I installed all the components and ran through the testing when I encountered that issue.  There's no feedback in any of the logs that the reason for the timeout is due to a missing network.json file.  That's not something you'd ever expect as being the root cause considering all tcp ports are open and responding on clc/cc/ufs etc...\r\nA simple log message hinting at this in the eucaconsole.log file would have saved 2-3 hours of testing followed by searching for what it could be.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dmccue","name":"dmccue","key":"dmccue","accountId":"557058:ed3db3e7-1778-45e1-a9ca-da74c4afc860","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/c6d14a696c9a8cbab9b66ec225f8bc8e?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"David McCue","active":true,"timeZone":"Europe/London"},"created":"2016-03-23T07:04:43.308-0500","updated":"2016-03-23T07:04:43.308-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/40871/comment/100607","id":"100607","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"body":"My big question is, since the console has no idea why it can't reach the CLC, should we mention the network.json file anyway?\r\nWhat wasn't explained, is the cloud expected to be functional without the network.json configured? If that isn't configured and we expect some things to work, what still isn't working?","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=dkavanagh","name":"dkavanagh","key":"dkavanagh","accountId":"557058:03be2b52-89f7-4c8f-8e00-e2a60a3e7a5e","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/e04f1add3c50bf8a7d1676b60b8ba337?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Ddkavanagh%26avatarId%3D10202%26noRedirect%3Dtrue"},"displayName":"David Kavanagh","active":false,"timeZone":"America/New_York"},"created":"2016-03-23T07:10:55.298-0500","updated":"2016-03-23T07:10:55.298-0500"}],"maxResults":11,"total":11,"startAt":0},"votes":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/GUI-2442/votes","votes":0,"hasVoted":false},"customfield_11808":null,"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]}}}